### PR TITLE
Libraries: Match output type and variable type

### PR DESCRIPTION
**Pull request by muramura**
I was detected with the cppcheck command that there was no output type and variable type.
I think that matching the type will increase the authenticity of the message.

### CODE REVIEW

**Comment by peterbarker** [comment]
Looks like this should have been merged and wasn't.  @muramura could you rebase, please?

File: libraries/AP_Scripting/lua/src/luac.c [hunk 1]
```diff
@@ -435,7 +435,7 @@ static void PrintDebug(const Proto* f)
  printf("upvalues (%d) for %p:\n",n,VOID(f));
  for (i=0; i<n; i++)
  {
-  printf("\t%d\t%s\t%d\t%d\n",
+  printf("\t%d\t%s\t%u\t%u\n",
```
Comment (WickedShell) [inline libraries/AP_Scripting/lua/src/luac.c]: Remove the change here, this is external code dependencies, and I'd rather not add conflicts.

**Comment by peterbarker** [comment]
@muramura this hasn't been rebased.

**Comment by tridge** [comment]
the RC decoders in AP_HAL/utility are going to be removed and replaced with AP_RCProtocol

**Comment by khancyr** [comment]
rebase, drop tabs change, address comment

**Pull request merged**